builtin/logical/pki/path_config_urls.go

```diff
@@ -46,9 +47,45 @@ set on all PR Secondary clusters.`,
 		Operations: map[logical.Operation]framework.OperationHandler{
 			logical.UpdateOperation: &framework.PathOperation{
 				Callback: b.pathWriteURL,
+				Responses: map[int][]framework.Response{
+					http.StatusNoContent: {{
+						Description: "No Content",
```

Comment:
This should now have content as of #18199

Reply:
Thank you!

Reply:
@AnPucel I think this holds globally now actually, almost every single `Update/Create` endpoint, especially for configs, will now return the same structure as Read, just so operators know if something they changed was correct or not.

Reply:
ahh. I see. Yes. I think I had that originally and saw that the responses being returned matched the Reads. I can update this.

Reply:
I think I got to all of them